src/psyclone/dynamo0p3.py

```diff
@@ -7452,137 +7451,136 @@ def gen_code(self, parent):
-#        self._add_dino_scalar(ndf_name)
-#
-#    def fs_compulsory_field(self, function_space):
-#        '''get dino to output compulsory arguments if there is a field on this
-#        function space'''
-#        undf_name = function_space.undf_name
-#        self._add_dino_scalar(undf_name)
-#
-#    def basis(self, function_space):
-#        '''get dino to output basis function information for the function
-#        space'''
-#        # TBD
-#        pass
-#
-#    def diff_basis(self, function_space):
-#        '''get dino to output differential basis function information for the
-#        function space'''
-#        # TBD
-#         pass
-#
-#    def orientation(self, function_space):
-#        '''get dino to output orientation information for the function
-#        space'''
-#        # TBD
-#        pass
-#
-#    def field_bcs_kernel(self, function_space):
-#        '''get dino to output any boundary_dofs information for bc_kernel'''
-#        # TBD
-#        pass
-#
-#    def quad_rule(self):
-#        '''get dino to output qr information '''
-#        # TBD
-#        pass
-#
-#    def generate(self):
-#        '''perform any additional actions before and after kernel
-#        argument-list based generation'''
-#        self._parent.add(CommentGen(self._parent, " dino output start"),
-#                         position=["before", self._position])
-#        scalar_comment = CommentGen(self._parent, " dino scalars")
-#        self._parent.add(scalar_comment,
-#                         position=["before", self._position])
-#        array_comment = CommentGen(self._parent, " dino arrays")
-#        self._parent.add(array_comment,
-#                         position=["before", self._position])
-#        self._scalar_position = scalar_comment.root
-#        self._array_position = array_comment.root
-#        self._parent.add(CommentGen(self._parent, " dino output end"),
-#                         position=["before", self._position])
-#        self._parent.add(CommentGen(self._parent, ""),
-#                         position=["before", self._position])
-#        ArgOrdering.generate(self)
-#
-#    def _add_dino_scalar(self, name):
-#        ''' add a dino output call for a scalar variable '''
-#        self._parent.add(CallGen(self._parent, name="dino%output_scalar",
-#                                 args=[name]),
-#                         position=["after", self._scalar_position])
-#
-#    def _add_dino_array(self, name):
-#        ''' add a dino output call for an array variable '''
-#        self._parent.add(CallGen(self._parent, name="dino%output_array",
-#                                 args=[name]),
-#                         position=["after", self._array_position])
+    def get_kernel_schedule(self):
+        '''Returns an LFRic PSyIR Schedule representing the kernel code. The
+        base class creates the PSyIR schedule on first invocation
+        which is then transformed to an LFRic PSyIR Schedule here. The
```

Comment:
It doesn't do any transformation yet - just validation checks. Perhaps a TODO?

Reply:
Good point. I wrote it for what I wanted it to do, not what it does. I've changed the wording.